sql/2_0_0-to-2_0_1_upgrade.sql

```diff
@@ -117,3 +117,16 @@ ALTER TABLE `users` DROP `fullscreen_role`;
 DROP TABLE IF EXISTS `menu_trees`;
 
 DROP TABLE IF EXISTS `menu_entries`;
+
+--Appointment Cancellation Reasons
+#IfNotRow2D list_options list_id lists option_id cancellation_reasons
+INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES (‘lists’,’cancellation_reasons’,’Cancellation Reasons’, 1,0);
+INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES (‘cancellation_reasons’,’1’,’No reason given’,5,0);
+INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES (‘cancellation_reasons’,’2’,’Work’,10,0);
+INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES (‘cancellation_reasons’,’3’,’Sick’,20,0);
+INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES (‘cancellation_reasons’,’4’,’Weather’,25,0);
+#EndIf
+
+#IfColumn reason patient_tracker_element
+ALTER TABLE `patient_tracker_element` ADD `reason` VARCHAR(1000) NOT NULL AFTER `user`;
```

Comment:
Change this field to varchar(255) or text. I don't think anyone is going to do 255 characters much less 1000 but text will allow them to write a book.